plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeMetadata.java

```diff
@@ -2446,6 +2543,26 @@ private CommitInfoEntry getCommitInfoEntry(
             long createdTime,
             String operation,
             long readVersion)
+    {
```

Comment:
I would probably just inline this one, rather than have two helper methods which just call a constructor